README.md

```diff
@@ -521,88 +521,55 @@ This is how to [extract the key manually](https://github.com/optskug/docs/blob/1
-        <ul>
-          <li>Stable <code>release-tizi</code> of sunnypilot is extensively tested by the secoc community.</li>
-          <li>Model switcher to easily switch between various AI models</li>
-          <li>NNLC: Big steering improvements for '21-23 RAV4 Prime and Sienna</li>
-          <li>sunnypilot complies with comma.ai's safety rules as accurately as possible.</li>
-          <li>‼️ If your device is C3 install <code>sunnypilot/staging-tici</code> instead. TICI hardware support at sunnypilot is slightly older than release but similarly capable at the moment.</li>
-        </ul>
-      </td>
-    </tr>
-    <tr>
-      <td>
-        <code>frogpilot.download</code> <br/>
-        (a.k.a. FrogPilot)<br/>
-        <a href="https://discord.com/invite/frogpilot">FrogPilot Discord</a>
-      </td>
-      <td>
-        <strong>Lat:</strong> Yes from upstream <br/>
-        <strong>MADS:</strong> Yes from community (AOL original author) <br/>
-        <strong>Long:</strong> Yes from community (from chrispypatt) <br/> <br/>
-        <ul>
-          <li>Uses an old AGNOS version. When downgrading, OP may get stuck in a registration loop. In this case, tap-tap-tap on the boot logo and reset the device to recover and then install again.</li>
-          <li>‼️ DO NOT RUN FROGPILOT DEEP STORAGE DELETE. It deletes your security key, and you have to run TSK Manager again. Run it only when you're selling the device.</li>
-        </ul>
-      </td>
-    </tr>
-</table>
+Unless otherwise noted, all forks support lat, MADS/AOL, and long.
+
+### openpilot
+
+URL: `commaani/nightly-dev`
+
+* Not a fork but an alternate branch from comma.ai with TSK support.
+* Install this if you need support from comma.ai company. They won't talk to you if you're on a fork.
+* Pre-compiled, so quick to install.
+* It has the most up-to-date changes, which is cool, but it could get unstable.
+* Supports lat and long, but no MADS/AOL.
+
+### SatoPilot
+
+URL: `alexandresato/personal3`
+
+* First fork to get long!
+* Very quick stop-and-go response.
+* `alexandresato/extract_secoc_key_btn` includes a TSK key extract button and is rebased with <code>personal3</code> often. The button is only for C3 & C3X, not for C4.
+
+### sunnypilot
+
+[URL](https://community.sunnypilot.ai/t/recommended-branch-installations/235)
+* C4: `dev.sunnypilot.ai`
+* C3X: `release.sunnypilot.ai`
+* C3: `install.sunnypilot.ai/staging-tici`
+
+Notes
+* Release branch of sunnypilot is extensively tested by the secoc community.
+* Model switcher to easily switch between various AI models.
+* NNLC: Big steering improvements for '21-23 RAV4 Prime and Sienna.
+* sunnypilot complies with comma.ai's safety rules as accurately as possible.
+* C3 branch (`statging-tici`) is slightly older than release but similarly capable at the moment.</li>
+* Discord: https://discord.gg/sunnypilot
+
+### FrogPilot
+
+URL: `frogpilot.download`
```

Comment:
What happens if you do this on a C4 at the moment? I am not sure of their C4 porting progress.

Reply:
Very good point. Let me check with them.

Reply:
[C4 not supported](https://discord.com/channels/1137853399715549214/1250897516413583421/1454912558321373316). Noted.